pkg/graph/schema.resolvers.go

```diff
@@ -796,9 +796,16 @@ func (r *subscriptionResolver) OnTaskListSectionLocksChanged(ctx context.Context
 	return resolvers.SubscribeTaskListSectionLockChanges(r.pubsub, modelPlanID, principal, ctx.Done())
 }
 
+// OnLockTaskListSectionContext is the resolver for the onLockTaskListSectionContext field.
+func (r *subscriptionResolver) OnLockTaskListSectionContext(ctx context.Context, modelPlanID uuid.UUID) (<-chan *model.TaskListSectionLockStatusChanged, error) {
+	principal := appcontext.Principal(ctx).ID()
+
+	return resolvers.OnLockTaskListSectionContext(r.pubsub, modelPlanID, principal, ctx.Done())
+}
+
 // Email is the resolver for the email field.
 func (r *userInfoResolver) Email(ctx context.Context, obj *models.UserInfo) (string, error) {
-	return string(obj.Email), nil
+	panic(fmt.Errorf("not implemented"))
```

Comment:
Not sure how this resolver got changed, but I think we should re-introduce the old behavior here!

Reply:
Not sure myself. Maybe a merge issue. Looking into it.